Cargo.toml

```diff
@@ -299,6 +299,7 @@ unix-archive = ".tar.gz"
 # Target platforms to build apps for (Rust target-triple syntax)
 targets = [
   "aarch64-apple-darwin",
+  "aarch64-pc-windows-msvc",
```

Comment:
...right yeah this shouldn't need a cargo-dist init...

Reply:
i did run generate or something